### PR TITLE
Upgrade Mariadb v10 5 and Improve Apt sourcehandling

### PR DESCRIPTION
**Pull request by tomjn**
- commented out broken piconet apt mirror
 - integrated distro name into source files for cross-distro support
 - upgraded MariaDB
 - always copy sources so they're fresh and up to date

## Checks

* [x] I've updated the changelog.
* [x] I've tested this PR
* [x] This PR is for the `develop` branch not the `stable` branch.
* [x] This PR is complete and ready for review.

### CODE REVIEW

**Comment by tomjn** [comment]
Related to #2390

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ permalink: /docs/en-US/changelog/
 ### Enhancements
 
 * Improved site provisioning messages
+* MariaDB upgraded too v10.5
```
Comment (evertiro) [inline CHANGELOG.md:14]: @tomjn My only critique... incorrect spelling of "to" for this context

**Review by evertiro — CHANGES_REQUESTED** [review]
One tiny typo

File: CHANGELOG.md (hunk 1 above)
Comment (tomjn) [reply CHANGELOG.md]: I'm on butterfly key mechanisms :(

**Review by evertiro — APPROVED** [review]
Working for me on Focal

**Pull request merged**